### PR TITLE
chore(server): refactoring analyzer files to comply with resources standard

### PR DESCRIPTION
**Pull request by danielbdias**
This PR updates the analyzer files to comply with the resource standard.

### CODE REVIEW

File: server/executor/linter_runner.go [hunk 1]
```diff
@@ -8,7 +8,7 @@ import (
 
 	"github.com/kubeshop/tracetest/server/analytics"
 	"github.com/kubeshop/tracetest/server/linter"
-	linterResource "github.com/kubeshop/tracetest/server/linter/resource"
+	analyzerResource "github.com/kubeshop/tracetest/server/linter/analyzer"
```
Comment (schoren) [inline server/executor/linter_runner.go:11]: should this be just `analyzer`? is there a reason for the alias?

Comment (danielbdias) [reply server/executor/linter_runner.go:11]: Good catch! I'll update it.

File: server/app/app.go [hunk 2]
```diff
@@ -24,7 +24,7 @@ import (
 	httpServer "github.com/kubeshop/tracetest/server/http"
 	"github.com/kubeshop/tracetest/server/http/mappings"
 	"github.com/kubeshop/tracetest/server/http/websocket"
-	linterResource "github.com/kubeshop/tracetest/server/linter/resource"
+	analyzer "github.com/kubeshop/tracetest/server/linter/analyzer"
```
Comment (mathnogueira) [inline server/app/app.go:27]: no need to use the alias if the alias has the same name as the package. You can remove them all

Comment (danielbdias) [reply server/app/app.go]: Thanks! Also updating it.

File: server/app/facade.go [hunk 3]
```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/kubeshop/tracetest/server/executor"
 	"github.com/kubeshop/tracetest/server/executor/pollingprofile"
 	"github.com/kubeshop/tracetest/server/executor/trigger"
-	linterResource "github.com/kubeshop/tracetest/server/linter/resource"
+	analyzer "github.com/kubeshop/tracetest/server/linter/analyzer"
```
Comment (mathnogueira) [inline server/app/facade.go:11]:
```suggestion
	"github.com/kubeshop/tracetest/server/linter/analyzer"
```

File: server/linter/analyzer/analyzer_repository_test.go [hunk 4]
```diff
@@ -1,27 +1,27 @@
-package linter_resource_test
+package analyzer_test
 
 import (
 	"database/sql"
 	"testing"
 
 	"github.com/gorilla/mux"
-	linter_resource "github.com/kubeshop/tracetest/server/linter/resource"
+	analyzer "github.com/kubeshop/tracetest/server/linter/analyzer"
```
Comment (mathnogueira) [inline server/linter/analyzer/analyzer_repository_test.go:8]:
```suggestion
	"github.com/kubeshop/tracetest/server/linter/analyzer"
```

File: server/linter/linter.go [hunk 5]
```diff
@@ -4,10 +4,10 @@ import (
 	"context"
 	"fmt"
 
+	analyzer "github.com/kubeshop/tracetest/server/linter/analyzer"
```
Comment (mathnogueira) [inline server/linter/linter.go:7]:
```suggestion
	"github.com/kubeshop/tracetest/server/linter/analyzer"
```

**Pull request merged**